src/content/docs/ai-observability/advanced-customization/drop-sensitive-data.mdx

```diff
@@ -0,0 +1,306 @@
+                Event
+            </th>
+            <th>
+                Attribute, or column 
+            </th>
+            <th>
+                Description 
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                `LlmChatCompletionMessage`
+            </td>
+            <td>
+                `content`
+            </td>
+            <td>
+                Describes when your model generates text to be delivered to your end user. For example, a completion event is triggered when the model generates a text response containing the address. The final message appers as the value for the attribute `content`.
+            </td>
+        </tr>
+        <tr>
+            <td>
+                `LlmFeedbackMessage`
+            </td>
+            <td>
+                `messages`
+            </td>
+            <td>
+                INSERT_CELL_CONTENT
+            </td>
+        </tr>
+        <tr>
+            <td>
+                `LlmEmbedding`
+            </td>
+            <td>
+                `input`
+            </td>
+            <td>
+                Indicates when your model uses a vector to interpret words and sentences. It appears when the LLM processes the prompt for the address and again when the model generates the response. The data appears as the value for the attribute `input`. 
+            </td>
+        </tr>
+        <tr>
+            <td>
+                `LlmTool`
+            </td>
+            <td>
+                `input` / `output`
+            </td>
+            <td>
+                INSERT_CELL_CONTENT
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+A complete set of drop filters for a piece of sensitive information should include queries for four events: `LlmChatCompletionMessage`, `LlmFeedbackMessage`, `LlmEmbedding`, and `LlmTool`. Your drop filters correspond to event and column pairs in a given row, as seen in the table. 
+
+* In other words, the attribute `content` only appears in the `LlmChatCompletionMessage` event and not in the `LlmEmbedding` event. The attribute `messages` only appears in `LlmFeedbackMessage` but not in `LlmTool`. 
+* The exception to this rule is the attribute `input`, which appears in both `LlmEmbedding` and `LlmTool`.
+* In the case of `LlmTool`, you need to create queries to account for two attributes `input` and `output`. For example:
+
+  ```sql
+      SELECT output FROM LlmTool WHERE output RLIKE ^([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})$
+      ```
+  ```sql
+    SELECT input FROM LlmTool WHERE input RLIKE ^([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})\.([0-9]{1,3})$
+    ```
+
+### Regex
+
+Since the agent's default behavior is to capture all parts of event data before sending it to NRDB, you need to direct the agent to match sensitive information with regex. By targeting an attribute with regex, you can still capture the event itself without storing sensitive information in our databases. 
```

Comment:
> Since the agent's default behavior is to capture all parts of event data before sending it to NRDB

Not quite an accurate statement (see above as well). You could change the `NRDB` to `New Relic` and it would fix that part of the sentence.

> you need to direct the agent to match sensitive information with regex.

It is not the agent that evaluates the regex, but instead the ingest pipeline after the agent has delivered the message to New Relic.